docs/en/ingest-management/elastic-agent/example-kubernetes-fleet-managed-agent-helm.asciidoc

```diff
@@ -0,0 +1,152 @@
+[[example-kubernetes-fleet-managed-agent-helm]]
+= Example: Install {fleet}-managed {agent} on Kubernetes using Helm
+
+preview::[]
+
+This example demonstrates how to install {fleet}-managed {agent} on a Kubernetes system using a Helm chart, gather Kubernetes metrics and send them to an {es} cluster in {ecloud}, and then view visualizations of those metrics in {kib}.
+
+For an overview of the {agent} Helm chart and its benefits, refer to <<install-on-kubernetes-using-helm>>.
+
+This guide will take you through the following steps:
+
+* <<agent-fleet-managed-helm-example-install-agent>>
+* <<agent-fleet-managed-helm-example-install-integration>>
+* <<agent-fleet-managed-helm-example-tidy-up>>
+
+
+[discrete]
+[[agent-fleet-managed-helm-example-prereqs]]
+=== Prerequisites
+
+To get started, you need:
+
+* A local install of the link:https://helm.sh/[Helm] Kubernetes package manager.
+* An link:{ess-trial}[{ecloud}] hosted {es} cluster on version 8.16 or higher.
+* An active Kubernetes cluster.
+* A local clone of the link:https://github.com/elastic/elastic-agent[elastic/elastic-agent] GitHub repository.
```

Comment:
maybe here we should specify the `8.16` branch to make that the Helm chart is fully compatible with elastic-agent 8.16.0 version?! I know it is somewhat inconvenient for now but as soon as we release the Helm chart on the elastic registry this will become better 🙂

Reply:
Nice catch @pkoutsovasilis! I've updated as shown, and the repo link now points directly to the [8.16 branch](https://github.com/elastic/elastic-agent/tree/8.16):

![Screenshot 2024-10-31 at 11 28 40 AM](https://github.com/user-attachments/assets/6165a8e5-75cc-458c-b0be-8ff3ed8caae9)